support/reftest/reftest

```diff
@@ -0,0 +1,296 @@
+#!/usr/bin/env python
+"""
+
+Prepare reference test data for Exodus-lambda integration test.
+It requires a "data.yml" file specifying the reference test data.
+
+"""
+
+import sys
+import yaml
+import boto3
+import tempfile
+import requests
+import argparse
+
+from tqdm import tqdm
+from datetime import datetime
+
+
+class DBHandler:
+    def __init__(
+        self,
+        table,
+        access_id=None,
+        access_key=None,
+        session_token=None,
+        region_name=None,
+    ):
+        self.dynamodb = boto3.Session(
+            aws_access_key_id=access_id,
+            aws_secret_access_key=access_key,
+            aws_session_token=session_token,
+            region_name=region_name,
+        ).client("dynamodb")
+
+        self.table = table
+
+    def search_item_from_web_uri(self, web_uri):
+        return self.dynamodb.query(
+            TableName=self.table,
+            Limit=1,
+            ScanIndexForward=False,
+            KeyConditionExpression="web_uri = :u",
+            ExpressionAttributeValues={":u": {"S": web_uri}},
+        )
+
+    def search_object_key_from_web_uri(self, web_uri):
+        res = self.search_item_from_web_uri(web_uri)
+        if not res["Items"]:
+            return False
+        else:
+            return res["Items"][0]["object_key"]["S"]
```

Comment:
This isn't guaranteed to be accurate, as the web_uri can be shared by multiple items. So, while this returns **an** object_key, we can't say it's the same file that was downloaded from the CDN.